src/components/contact-us/contact-us-controller.ts

```diff
@@ -404,6 +404,7 @@ export function contactUsQuestionsFormPostToSmartAgent(
         ticketIdentifier: ticketIdentifier,
         userAgent: req.get("User-Agent"),
         appErrorCode: getAppErrorCode(req.body.appErrorCode),
+        country: req.body.country,
```

Comment:
Hi @VladGavrilet, I tested the Zendesk submission earlier today and found that the`country` field wasn't coming through. Having looked into it, it seems this is because it's only been added to the SmartAgent service call (here) and not to the Zendesk service call. This line also need to be added there.